### PR TITLE
2020 redesign dev

### PR DESCRIPTION
**Pull request by devagrawal09**

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -63,9 +61,9 @@ export default () => (
       </div>
     </section>
     <section id="faq" className="py-20 lg:pb-40 lg:pt-48">
```
[the remark this answers is not in the document]
Comment (devagrawal09) [reply src/pages/index.js:63]: this one is @Kunalkrk

File: src/pages/register.tsx [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+                        />
+                      </div>
+                      <div className="md:items-center py-20 lg:pb-20 lg:pt-10">
+                        <button
+                          className="shadow bg-purple-500 hover:bg-purple-400 focus:shadow-outline focus:outline-none text-white font-bold py-2 px-4 rounded"
+                          type="submit"
+                          disabled={submitting}
+                        >
+                          
+                          {submitting ? <FaSpinner /> : `Register`}
```
[the remark this answers is not in the document]
Comment (devagrawal09) [reply src/pages/register.tsx:302]: What do you mean?

File: src/pages/index.js [hunk 3]
```diff
@@ -1,30 +1,25 @@
 import React from 'react';
+import { Link } from 'gatsby';
 import Button from '../components/Button';
 import Card from '../components/Card';
-import LabelText from '../components/LabelText';
 import Layout from '../components/layout/Layout';
-import ScrollTopButton from '../components/ScrollTopButton';
-// import StatsBox from '../components/StatsBox';
 import HeroImage from '../svg/HeroImage';
```
[the remark this answers is not in the document]
Comment (devagrawal09) [reply src/pages/index.js:6]: I thought you were going to work on this section? (as per our last meeting) Also, I don't have the image for this

File: src/pages/register.tsx [hunk 4]
```diff
@@ -0,0 +1,317 @@
+import React, { useState } from 'react';
```
[the remark this answers is not in the document]
Comment (devagrawal09) [reply src/pages/register.tsx:1]: I did, but the page is not long enough for that button to show up, so I removed it

File: src/components/ScrollTopButton.js [hunk 5]
```diff
@@ -1,19 +1,25 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (devagrawal09) [reply src/components/ScrollTopButton.js:1]: How? it's an image, I'll need two separate images for that to work

File: src/pages/register.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (omaralsayed) [reply src/pages/register.tsx:302]: Button colors do not match other buttons on website

File: src/pages/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (omaralsayed) [reply src/pages/index.js:6]: The image is in the shared folder in Slack. Sorry about that, I thought that you wanted to take care of all branding and styling, I'll update this later today!

File: src/components/ScrollTopButton.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (omaralsayed) [reply src/components/ScrollTopButton.js:1]: Yep, that's correct. There are two different images in the shared folder in Slack.

**Pull request merged**